readme.md

```diff
@@ -760,6 +760,7 @@
 
 - [Slack](https://github.com/matiassingers/awesome-slack#readme) - Team collaboration.
 	- [Communities](https://github.com/filipelinhares/awesome-slack#readme)
+	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps-for-developers#readme) - Slack integrations to help developers and tech teams be more productive.
```

Comment:
```suggestion
	- [Slack Apps for Developers](https://github.com/axolo-co/awesome-slack-apps-for-developers#readme) - Slack integrations to help developers and tech teams be more productive.
```

And the list heading should match this capitalisation.

Reply:
Jo